src/routes/Results.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Button, Icon, Pane, Paneset, PaneMenu, IconButton } from '@folio/stripes-components';
```

Comment:
Please change the import to `@folio/stripes/components` as `@folio/stripes` is now preferred option for that.